src/main/resources/static/js/register.js

```diff
@@ -5,13 +5,15 @@ const maxAge = 99;
 $(function() {
 
 	let today = new Date();
-	let startDate = new Date(today.setFullYear(today.getFullYear() - minAge)).toISOString().split('T')[0];
```

Comment:
This line calls `todat.setFullYear()`, which changes the return value of `today.getFullYear()` in the next line, therefore the `endDate` is offset by `minAge`.